web/src/components/PageHeader/index.js

```diff
@@ -19,7 +23,28 @@ function PageHeader() {
           breadcrumbName: route.title,
         };
       } else {
-        return null;
+        if (pathSnippets.includes('edit') && !state[entity].loading) {
```

Comment:
@Bhumi101097 Instead of checking entities loading state check we whether we have that entity or not.